Resources/DNNCE_nl-NL.dnn

```diff
@@ -1,985 +1,987 @@
+						</languageFile>
+					</languageFiles>
+				</component>
+			</components>
+		</package>
+		<package name="DDRMenu_nl-NL" type="ExtensionLanguagePack" version="09.02.00">
+			<friendlyName>DDR Menu Nederlands (Nederland)</friendlyName>
+			<description><![CDATA[Dutch (Netherlands) language pack for DDR Menu.<br />This translation of DNN into Dutch (Netherlands) was done by <a href="http://www.xcess.nl">XCESS expertise center b.v.</a>. All copyrights for the original and translated language are owned by DNN Corp.]]></description>
+			<iconFile />
+			<owner>
+				<name>DNN Corporation</name>
+				<organization>DNN Corporation</organization>
+				<url>www.dnnsoftware.com</url>
+				<email>support@dnnsoftware.com</email>
+			</owner>
+			<license>DNN - http://www.dnnsoftware.com&lt;br&gt;Copyright (c) 2002-2018&lt;br&gt;by DNN Corp.&lt;br&gt;&lt;br&gt;Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:&lt;br&gt;&lt;br&gt;The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.&lt;br&gt;&lt;br&gt;THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.</license>
+			<releaseNotes />
+			<components>
+				<component type="ExtensionLanguage">
+					<languageFiles>
+						<code>nl-NL</code>
+						<package>DDRMenu</package>
+						<languageFile>
+							<path>DesktopModules\DDRMenu\App_LocalResources</path>
+							<name>MenuSettings.ascx.nl-NL.resx</name>
+						</languageFile>
+						<languageFile>
+							<path>DesktopModules\DDRMenu\App_LocalResources</path>
+							<name>MenuView.ascx.nl-NL.resx</name>
+						</languageFile>
+					</languageFiles>
+				</component>
+			</components>
+		</package>
+		<package name="DigitalAssetsManagement_nl-NL" type="ExtensionLanguagePack" version="09.02.00">
+			<friendlyName>Digital Asset Management Nederlands (Nederland)</friendlyName>
+			<description><![CDATA[Dutch (Netherlands) language pack for Digital Asset Management.<br />This translation of DNN into Dutch (Netherlands) was done by <a href="http://www.xcess.nl">XCESS expertise center b.v.</a>. All copyrights for the original and translated language are owned by DNN Corp.]]></description>
```

Comment:
I did not do anything on the license text, this is what is was in the first place

Reply:
Yeah I think that got missed and only the first node was updated for current status. I gonna approve this but it would be nice to update it in another PR...